packages/api-types/src/index.ts

```diff
@@ -69,6 +69,29 @@ export type PostLanguageRequestBody = Language;
 
 export type PatchLanguageRequestBody = Partial<Omit<Language, 'code'>>;
 
+export const LanguageRole = makeEnum({
```

Comment:
This doesn't contain viewer?

Reply:
The model is a little different on the API side. If you have the viewer role, then you will be listed among the members of the language in the language members request. All users that are added the language get the viewer role. The role field in the api routes then only delineates the extra roles you have on the language.

I'm open to feedback if you think this structure is confusing. I was trying to simplify the database model a bit so that there is just a table of language roles (columns: language ID, user ID, role). If you keep track of members and roles separately you have another table that has similar data (columns: language ID, user ID). That seemed unnecessary.

Reply:
Can a user have one role or multiple? Does the Administrator role automatically encompass all parts of the translator role?

Reply:
Users can have multiple roles. Admin lets you manage users, translator lets you work with the translations